patch/series

```diff
@@ -140,7 +138,7 @@ Support-for-fullcone-nat.patch
 0092-platform-mellanox-Introduce-support-of-Nvidia-smart-.patch
 0093-hwmon-pmbus-Add-support-for-MPS-Multi-phase-mp2855-c.patch
 0094-i2c-asf-Introduce-MCTP-support-over-ASF-controller.patch
-0095-platform-mellanox-nvsw-sn2201-Add-support-for-new-sy.patch
+#0095-platform-mellanox-nvsw-sn2201-Add-support-for-new-sy.patch
```

Comment:
Why?

Reply:
There was a patch conflict, so I disabled it just for the purposes of getting the build working. This set of patches is managed by Mellanox, and they'll update these patches as needed for 6.1.123.